src/Autofac.Integration.AspNetCore.Multitenant/MultitenantRequestServicesMiddleware.cs

```diff
@@ -15,10 +15,8 @@ namespace Autofac.Integration.AspNetCore.Multitenant
     internal class MultitenantRequestServicesMiddleware
     {
         private readonly IHttpContextAccessor _contextAccessor;
-
-        private readonly RequestDelegate _next;
-
         private readonly IServiceProvider _serviceProvider;
+        private readonly RequestDelegate _next;
```

Comment:
Was this necessary or just a one-click fix side effect?

Reply:
Was mistakenly done. Changed it back now.